### ISSUE TITLE
Console d'admin : fixer la largeur du champ mail

### ISSUE THREAD

**Issue by andreconseil** [mission-apprentissage/voeux-affelnet#36]
Actuellement  trop étroit lorsqu'il y a une liste importante d'UAI (cf capture)

**Comment by andreconseil** [comment]
![Image](https://user-images.githubusercontent.com/33602876/169261083-6a29ce05-5d78-467a-9169-79264509e434.png)

**Comment by bguerout** [comment]
Oui c'était en quickwin pour avoir la donnée affichée.
Je pense qu'il faudrait faire une page ou une popup CFA si on veut afficher proprement ce genre d'informations.

**Comment by QuentinPetel** [comment]
Est-ce que ça irait comme ça : 

![Image](https://user-images.githubusercontent.com/16292377/169330714-b5736ee6-da74-4788-872e-5e6659314778.png)

**Comment by andreconseil** [comment]
Oui, par contre attention tu n'as visiblement pas en local la colonne "établissements" (c'est justement celle qui contribue le + à réduire la largeur du champ email)

**Comment by andreconseil** [comment]
OK j'avais pas vu. OK comme ça.
